### PR TITLE
Clarification about SLURM names and daemon update

### PR DESCRIPTION
**Pull request by middelkoopt**
Clarification about SLURM names and daemon update.  See issue #1823

### CODE REVIEW

File: docs/recipes/install/common/install_slurm.tex [hunk 1]
```diff
@@ -36,10 +36,10 @@
   for compute nodes under its control. In particular, three configuration
   parameters combine to define consumable compute resources: {\em Sockets},
   {\em CoresPerSocket}, and {\em ThreadsPerCore}. The default configuration
-  file provided via \OHPC{} assumes dual-socket, 8 cores per socket, and two
+  file provided via \OHPC{} assumes the nodes are named c1-c4 and are dual-socket, 8 cores per socket, and two
   threads per core for this 4-node example. If this does not reflect your local
   hardware, please update the configuration file at
-  \path{/etc/slurm/slurm.conf} accordingly to match your particular hardware.
+  \path{/etc/slurm/slurm.conf} accordingly to match your nodes names and particular hardware. Be sure to run \texttt{scontrol reconfigure} to notify SLURM of the changes.
```
Comment (martin-g) [inline docs/recipes/install/common/install_slurm.tex]: nit: the line above is too long compared to the rest. Consider moving the new sentence on its own new line.

Comment (middelkoopt) [reply docs/recipes/install/common/install_slurm.tex]: Paragraph wrapped with new commit.

**Comment by middelkoopt** [comment]
Squashed fixup commit and re-pushed.

**Comment by martin-g** [comment]
Please rebase to latest 2.x to fix the CI failure.

**Comment by middelkoopt** [comment]
Rebased against 2.x

**Comment by adrianreber** [comment]
I would like to see this on 3.x. I leave it up to you if you also want it on 2.x.

**Comment by middelkoopt** [comment]
Rebased against 2.x. Reapplied against 3.x in PR #1941.

**Pull request merged**